numpy/f2py/_backends/src/meson.build.src

```diff
@@ -0,0 +1,40 @@
+project('${modulename}',
```

Comment:
Besides the file name, it'd be better not to have an extra subdir here with only a single file in it. You can move it up.